### ISSUE TITLE
Password form without style?

### ISSUE THREAD

**Issue by alekso87** [ether/etherpad-lite#3947]
Hi everyone! I'm testing pads with a password, I came across a form in which, as for me, there are no styles.

theme - colibris

<img width="870" alt="Screenshot at Apr 27 15-55-48" src="https://user-images.githubusercontent.com/25965602/80377379-e1f1fa00-88a3-11ea-9f15-14c7aef763ec.png">

**Comment by JohnMcLear** [comment]
Oh I thought that whole thing got dropped/removed.  I wouldn't use the internal Etherpad password model and instead use a proper Auth provider.

I think we are going to remove this password option soon.  Do you need it?

**Comment by JohnMcLear** [comment]
Cc @seballot

**Comment by seballot** [comment]
I was not even aware of this feature !

**Comment by alekso87** [comment]
> and instead use a proper Auth provider.

What do you mean?

> Do you need it?

Today I'll sit down to rethink the model of my project so as not to use a password. A little later I will give an answer.

**Comment by JohnMcLear** [comment]
Well Etherpad is not a good auth provider.  Intentionally...

Search here for "auth" https://static.etherpad.org/ -- some examples:
ldap, oauth..  There are hundreds and it's best to just allow Etherpad to use whichever suits your entire platform best.  

TLDR; author auth mechanisms are mature/secure/well maintained..  It makes sense to just offload auth external to Etherpad and only have Etherpad handle the sessions.

**Comment by alekso87** [comment]
@JohnMcLear It's good that I raised this question on time :) Thanks for the clarification! As I understand it, in the future, the function of creating a password will be deleted from the project?

**Comment by JohnMcLear** [comment]
We have a proposal open to make this change.  It isn't concluded yet but I would say there is a 80% chance we will go that way.  Is that okay?  Obviously if it's critical for you(or others) we will keep it in but I think most devs realize having our own auth mechanism is a pretty wide vector to attack.

**Comment by alekso87** [comment]
I agree about the security. It’s better to let everyone take care of authentication, it’s more difficult to break through the defense and apply the attack en masse.

**Comment by CornMaster** [comment]
I'm interested in password protected pads, but I'm not concerned if the functionality exists within core or is implemented through a plugin.  I can see the benefits of moving that to a plugin.

My lightweight use case involves a system that would be used by a team to collaborate with outsiders.  Creating and managing accounts for team is easy, but for the public group would be a lot of work, and additionally we wouldn't want to enable open signup.  So password protected pads created by the team and shared with clients would be a suitable workflow for us.

I've looked through the plugin database, and haven't come across one yet.  Is there an existing plugin that enables passwords on pads?  If not, is anyone aware of someone developing such a plugin and/or would the existing functionality in core be spun off into a plugin?  I may be able to lend some support developing/testing such a plugin.

Thanks

**Comment by JohnMcLear** [comment]
https://static.etherpad.org/

Search for mypads

**Comment by working-name** [comment]
@CornMaster did anything come about from your efforts? Looking for something simple as well, just an "enter password" prompt would be perfect. I have bumped into several recommendations to no longer use etherpad's internal password feature by @JohnMcLear but unfortunately no plugin that I can find can accomplish that simple task. So if I were @JohnMcLear I wouldn't knock that simple password protection just to avoid having etherpad "handle authentication". I think password protecting pads and authentication are 2 very distinct things and are not mutually exclusive.

MyPads specifically is an alien, outside of etherpad, does not feel like an integrated plugin. It's a standalone management of the group pads feature but clunky, and unfortunately has nothing to do with etherpad's design language. It also does not handle password protecting pads, it just forces an account creation model instead, and besides having to manually lead the user to a weird URL to log in rather than hooking into etherpad while loading the specific pad, it also fails to redirect the user back to the initially requested pad after authentication. It loads up the "dashboard" or whatever they call that, which unfortunately is confusing and of 0 use to a regular user who just wanted to type on a pad with other people. Not a viable option in my opinion, at least not for password protecting a pad or making it easy for a regular user to work with non-public pads.
